statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1083,7 +1083,7 @@ def plotsim(self, steps=None, offset=None, seed=None):
         """Deprecated.  Use plot_sim."""
         import warnings
         warnings.warn("plotsim is deprecated and will be removed in 0.11.0. "
-                      "Use plot_sim.", DeprecationWarning)
+                      "Use plot_sim.", FutureWarning)
```

Comment:
Isn't DeprecationWarning in these cases more appropriate?

Reply:
There is now guidance from python.org: Future for users, Deprecation for developers.  So Future is almost always the right warning.  Older posts seemed to indicate Future was for this is definitely going away and deprecation is for things that may go away but shouldn't be used because there is a better way now.